module/VuFind/src/VuFind/Content/ObalkyKnihService.php

```diff
@@ -176,7 +176,15 @@ protected function getFromService(array $ids): ?\stdClass
     {
         $param = "multi";
         $query = [];
-        $isbn = isset($ids['isbn']) ? $ids['isbn']->get13() : null;
+        $isbn = null;
+        if (!empty($ids['isbns'])) {
+            $isbn = array_map(
+                function ($isbn) {
+                    return $isbn->get13();
+                },
+                $ids['isbns']
+            );
+        }
```

Comment:
Is it worth adding an `elseif (!empty($ids['isbn'])) {` case? I realize that that should never happen post-refactoring, but it might be useful to maintain that part of the API in case this plugin is ever being used in a different context in custom code.

Reply:
I've done it.